src/app/layouts/base-layout/components/footer/about-list.tsx

```diff
@@ -25,7 +25,7 @@ export const AboutList = () => {
   return (
     <div className="about-list">
```

Comment:
Maybe use <figure>?

Reply:
in this file I replaced the logo usage, the rest may need to be refactored as part of another task?

Reply:
I reviewed the code and used the figure tag, but in the logo component

Reply:
> Maybe use figure tag?

Could you provide sources showing that using the `<figure>` tag is considered best practice for our case?

Reply:
I don't know if Nikita can answer. But since our logo is a separate and independent component, I thought that it could be wrapped in a tag in the component itself.
[MDN-Figure](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure)

Reply:
It was decided to leave the div. Fixed